index.html

```diff
@@ -3098,7 +3098,113 @@ <h1>Speakers</h1>
+                                            <a target="default" href="https://twitter.com/TRakeshSREC"><i class="icon social_twitter"></i></a>
+
+                                        </div>
+                                    </div>
+                                </div>
+                                <span class="speaker-name">Rakesh Tadisetty</span>
+                                <span>Developer</span>
+                                <span>Ingenious Solutions Software</span>
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Divyam madaan" src="img/Divyammadaan.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://divyam3897.github.io/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://twitter.com/divyam3897"><i class="icon social_twitter"></i></a>
+
+                                            <a target="default" href="https://github.com/divyam3897"><i class="icon fa fa-github"></i></a>
+                                        </div>
+                                    </div>
+                                </div>
+                                <span class="speaker-name">Divyam madaan</span>
+                                <span>Software Developer</span>
+                                <span>KDE</span>
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Salman Rahman Desh" src="img/Salman_Rahman_Desh.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="http://srdesh.info/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://twitter.com/Salman_Desh"><i class="icon social_twitter"></i></a>
+                                            <a target="default" href="https://www.linkedin.com/in/salmanrahmandesh/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://github.com/srdesh"><i class="icon fa fa-github"></i></a>
+                                        </div>
```

Comment:
Please add Facebook link also.

Reply:
Thank you for the suggestions I will implement them now